python/pyspark/sql/tests/connect/test_utils.py

```diff
@@ -0,0 +1,19 @@
+from pyspark.testing.connectutils import ReusedConnectTestCase
```

Comment:
`pyspark.sql.tests.connect.test_utils` has to be put in `dev/sparktestsupport/modules.py` so it actually runs the test.